c/alloc.c

```diff
@@ -410,7 +422,9 @@ char *safeMalloc(int size, char *site){
       abend(MALLOC_ABEND_CODE, MALLOC_ABEND_REASON);
     }
 #endif
-    printf("MALLOC failed, got NULL for size %d at site %s\n",size, site);
+    if (MALLOC_TRACE_LEVEL >= 1){
+      printf("MALLOC failed, got NULL for size %d at site %s\n",size, site);
```

Comment:
This just may not be needed at all, because you will know success by getting a return of NULL.